apis/kueue/v1beta1/workload_types.go

```diff
@@ -75,6 +75,15 @@ type WorkloadSpec struct {
 	// Defaults to true
 	// +kubebuilder:default=true
 	Active *bool `json:"active,omitempty"`
+
+	// MaximumExecutionTimeSeconds if provided, determines the maximum time, in seconds,
```

Comment:
```suggestion
	// maximumExecutionTimeSeconds if provided, determines the maximum time, in seconds,
```

Let us use the YAML name so that users can understand it easily.

Reply:
done, but we should come up wit a convention for this, the styles are mixed :).